AllReadyApp/Web-App/AllReady/DataAccess/AllReadyDataAccessEF7.Itinerary.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Extensions.Internal;
+
+namespace AllReady.Models
+{
+    public partial class AllReadyDataAccessEF7 : IAllReadyDataAccess
```

Comment:
We are actually moving away from the `AllReadyDataAccessEF7` class usage. Have you seen the work we're doing with handlers? I'd be happy to point you at some resources or Skype up with you to walk you though implementation if you like.